builtin/game/item_entity.lua

```diff
@@ -36,6 +36,9 @@ core.register_entity(":__builtin:item", {
 	itemstring = '',
 	physical_state = true,
 	age = 0,
+	-- The items are mergeable as long as this property is not false,
+	-- so we use nil to save a few bytes unless it will be set.
+	mergeable = nil,
```

Comment:
@ShadowNinja I'd rather not have a asymmetric serialization process. Because the serialization function would turn `true` into `nil` and the deserialization function would need to turn `nil` into `true`. That means that if the user `nil`s the value of the `mergeable` field, it will behave like it is set to false, unless it has been serializaed/deserialized ones.

Arrr, can we forget about the memory overhead and simply implement it straight forward without any magic in between? Alternately, we can do functions which hide all the complicated stuff.